plugins/org.jboss.reddeer.eclipse/src/org/jboss/reddeer/eclipse/debug/ui/launchConfigurations/LaunchConfigurationsDialog.java

```diff
@@ -206,6 +206,12 @@ public String errorMessageWhile() {
 		public String errorMessageUntil() {
 			return "Tree does not has focus or TreeItem '"+item.getText()+"' is not selected";
 		}
+
+		@Override
```

Comment:
definitely. We didnt have AbstractWaitCondition in the past so there might be quite a few wait conditions which are just implementing WaitCondition instead of extending AbstractWC.